src/app/qgsdecorationcopyrightdialog.cpp

```diff
@@ -49,6 +49,12 @@ QgsDecorationCopyrightDialog::QgsDecorationCopyrightDialog( QgsDecorationCopyrig
   cboPlacement->setCurrentIndex( cboPlacement->findData( mDeco.placement() ) );
   spnHorizontal->setValue( mDeco.mMarginHorizontal );
   spnVertical->setValue( mDeco.mMarginVertical );
+  QStringList mySelectionUnitItems;
```

Comment:
If the units are changed to QgsSymbolV2::OutputUnit then this can be simplified (and made safer) by using a method like:
https://github.com/qgis/QGIS/blob/master/src/gui/effects/qgspainteffectwidget.cpp#L257

Then setting the current unit is done like https://github.com/qgis/QGIS/blob/master/src/gui/effects/qgspainteffectwidget.cpp#L283 and retrieving like https://github.com/qgis/QGIS/blob/master/src/gui/effects/qgspainteffectwidget.cpp#L343
(ignore the "mapUnitScale" code - that's not relevant for this, as map units won't be a choice in the widget).